### DISCUSSION TITLE
[Question] Progressive Web Application and Capacitor integration

### DISCUSSION THREAD

**Discussion by mattveraldi** [remix-run/remix#2743] [Q&A] [answered]
Hi, is it possible to build a Progressive Web Application and to integrate Capacitor in order to get an installable apk of the web app?

**Comment by sergiodxa** [comment, marked as answer]
If you add a manifest.json, a SW and any other requirement to make your web app a PWA you should be able to create a PWA easily, I’m not sure what’s needed for capacitor to be used.

**Comment by mattveraldi** [comment]
@sergiodxa thank you

**Comment by adrm** [comment]
Capacitor just released some starting templates for remix projects with capacitor https://github.com/ionic-team/capacitor-remix-templates
